### DISCUSSION TITLE
Roundtrip possible?

### DISCUSSION THREAD

**Discussion by AllTradesz** [Synthoid/ExportSheetData#143] [Q&A] [answered]
Can this tool be used in a roundtrip manner? That is, export a Sheet (may include multiple worksheets), then later upload the exported file into a new Sheet in someone else's Google account with all styles and settings intact, including of course cell formulae, cell notes, column sizes, cell alignments, etc.

Thanks!

**Comment by Synthoid** [comment, marked as answer]
Sorry for the incredibly late response. Not sure why I didn't get notifications for discussions until today. That sort of round trip operation is technically possible, but not out of the box. ESD doesn't import JSON/XML into spreadsheets, it can only export data. This is primarily because data import is subject to all kinds of specific rules based on the data and desired format. Additionally, things like styles (I assume you mean font selection, tinting, etc) and cell formulas are lost when exporting JSON as only cell values are exported into a standard text format that doesn't contain style data.

Unfortunately, that means that what you are asking is not possible at this time and likely will not be part of ESD's functionality due to the incredibly specific nature of the feature.
